servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -106,9 +107,8 @@ public ReferencesResponse getAllReferences(ReferencesParams params) {
         params.isFetchAdditionalInfo()
             ? GetNamedRefsParams.builder()
                 .baseReference(BranchName.of(this.getConfig().getDefaultBranch()))
-                .isComputeAheadBehind(true)
-                .isRetrieveCommitMetaForHead(true)
-                .isComputeCommonAncestor(true)
+                .branchRetrieveOptions(RetrieveOptions.BASE_REFERENCE_RELATED_AND_COMMIT_META)
+                .tagRetrieveOptions(RetrieveOptions.COMMIT_META)
```

Comment:
could we also set this to `tagRetrieveOptions(RetrieveOptions.BARE)` since we currently don't show any metadata for Tags?